code/modules/food_and_drinks/kitchen_machinery/smartfridge.dm

```diff
@@ -494,6 +493,91 @@
 		new /obj/item/stack/sheet/metal(loc, 4)
 	qdel(src)
 
+/**
+  * # Food and Drink Cart
+  *
+  * Secure variant of the [Food Cart][/obj/machinery/smartfridge/foodcart].
+  */
+/obj/machinery/smartfridge/secure/foodcart
+	name = "food and drink cart"
+	desc = "A portable cart for hawking your food and drink wares around the station. This one seems to have an ID scanner on it."
+	icon = 'icons/obj/kitchen.dmi'
+	icon_state = "foodcart"
+	anchored = FALSE
+	interact_offline = TRUE
+	power_state = NO_POWER_USE
+	visible_contents = FALSE
+	face_while_pulling = FALSE
+	silicon_controllable = FALSE
+
+/obj/machinery/smartfridge/secure/foodcart/Initialize(mapload)
+	. = ..()
+	req_one_access_txt = "[ACCESS_KITCHEN], [ACCESS_BAR]"
+	accepted_items_typecache = typecacheof(list(
+		/obj/item/food/snacks,
+		/obj/item/reagent_containers/drinks,
+		/obj/item/reagent_containers/condiment,
+	))
+
+/obj/machinery/smartfridge/secure/foodcart/attackby(obj/item/O, mob/user, params) // Anti-theft technology, inspired by the station nuke
+	if(istype(O, /obj/item/card/id))
+		if(!allowed(user))
+			to_chat(user, "<span class='warning'>Access denied.</span>")
+			return
+		if(emagged)
+			to_chat(user, "<span class='warning'>The cart is silent as a brief puff of black smoke rises from the ID scanner.</span>")
+			return
+		if(anchored)
+			to_chat(user, "<span class='warning'>You release the locking mechanism, unanchoring the cart.</span>")
+			anchored = FALSE
+			playsound(loc, 'sound/machines/boltsup.ogg', 30, 0, 3)
+			return
+		to_chat(user, "<span class='warning'>You swipe your ID over the scanner, and with a steely snap bolts slide out from underneath the cart, securely anchoring it on the floor.</span>")
```

Comment:
```suggestion
		to_chat(user, "<span class='warning'>You swipe your ID over the scanner, and with a hefty snap, bolts slide out from underneath the cart securely anchoring the cart to the floor.</span>")
```

Reply:
I intentionally used steely since it was used in the nuke bolting description - I don't mind changing it but just wanted to confirm

Reply:
Changed in latest commit.